clang-tools-extra/clang-doc/tool/ClangDocMain.cpp

```diff
@@ -49,6 +49,8 @@
 using namespace clang::ast_matchers;
 using namespace clang::tooling;
 using namespace clang;
+using DirIterator = llvm::sys::fs::directory_iterator;
```

Comment:
There is a` DirIterator` in LLVM's VFS, and it is confusing to see you reuse that name. Normally iterators are written `It` in LLVM. ``DirIt` is probably a better name. I don't think it needs to be at file scope either, since its basically used in 1 place. you can add the using statement w/in the function that needs it, since its just for clarification.

If you really want it at files scope, stop at `using llvm::sys::fs`